pl_examples/loop_examples/yielding_training_step.py

```diff
@@ -56,28 +55,25 @@ def __init__(self):
     def connect(self, **kwargs):
         raise NotImplementedError(f"{self.__class__.__name__} does not connect any child loops.")
 
-    def on_run_start(self, batch, optimizers, batch_idx):
-        super().on_run_start(batch, optimizers, batch_idx)
+    def on_run_start(self, optimizers, kwargs):
```

Comment:
shall we rather propagate it as `**kwargs` again?

Reply:
No because we want to pass the container which is what's modified per-loop.